recipe/meta.yaml

```diff
@@ -11,41 +11,54 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-
-  run:
-    - python >=3.6
-    - cycler
-    - databroker
-    - doct
-    - event-model >=1.10.0
-    - historydict
-    - ipython
-    - lmfit
-    - matplotlib
-    - msgpack-python
-    - msgpack-numpy
-    - numpy
-    - ophyd
-    - pyzmq
-    - super_state_machine
-    - toolz
-    - tqdm
-    - zict
+  number: 1
 
 test:
   imports:
     - bluesky
     - bluesky.plans
     - bluesky.callbacks
 
+outputs:
+- name: bluesky-base
+  build:
+    noarch: python
+    script: {{ PYTHON }} -m pip install . --no-deps -vv
+  requirements:
+    host:
+      - python >=3.6
+      - pip
+    run:
+      - python >=3.6
+      - cycler
+      - event-model >=1.10.0
+      - historydict
+      - msgpack-numpy
+      - msgpack-python
+      - numpy
+      - super_state_machine
+      - toolz
+      - tqdm >=4.44
+      - zict
+- name: bluesky
+  build:
+    noarch: python
+    script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
This seems wrong to me as now there is going to be two packages which claim the same files.  I would expect the bluesky package to be effectively code less (!!) and only have dependencies.